HACKING.adoc

```diff
@@ -132,6 +132,17 @@ independent and should not need further knowledge.
 link:otherlibs/[]:: External libraries such as `unix`, `threads`,
 `dynlink`, `str` and `bigarray`.
 
+While building the full reference manual is described in
+link:manual/README.md[]. But if you only modify the documentation comments
```

Comment:
This first sentence doesn't make sense.  Presumably you meant "Instructions for building the full reference manual are provided in ..." ?

Second sentence: "However" instead of "But" is better, I think.

Also it should be "in `.mli` files", not "of `.mli` files".